core/services/relay/evm/target_strategy.go

```diff
@@ -0,0 +1,230 @@
+package evm
```

Comment:
To improve readability and discoverability consider scoping this code to a package like `evm/writetarget` instead of root `evm`

Reply:
Doing so would lead to an import cycle